android/java/org/chromium/chrome/browser/crypto_wallet/web_ui/WebUiActivityType.java

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.crypto_wallet.web_ui;
+
+import android.app.Activity;
+
+import org.chromium.chrome.browser.crypto_wallet.util.Utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Type of WebUi Activity.
+ * @see Utils#openBuySendSwapActivity(Activity, WebUiActivityType).
+ */
+public enum WebUiActivityType {
+    BUY(0),
+    SEND(1),
+    SWAP(2);
+
+    private final int value;
+    private static final Map<Integer, WebUiActivityType> map = new HashMap<>();
+
+    WebUiActivityType(final int value) {
+        this.value = value;
+    }
+
+    static {
+        for (WebUiActivityType webUiActivityType : WebUiActivityType.values()) {
+            map.put(webUiActivityType.value, webUiActivityType);
+        }
+    }
+
+    public static WebUiActivityType valueOf(int activityType) {
+        return map.get(activityType);
+    }
+
+    public int getValue() {
+        return value;
+    }
```

Comment:
nit: why `map` and related methods are required? seems like can be removed. (and the name should be `sMap` for static though)

Reply:
Yeah, I was skeptical at first, but didn't invest much time into that. If you don't know either, I'll try to remove it as it seems unneeded. 
Thanks!

Reply:
I believe that we don't need it so just `create enum without any explicit value`